src/ts/lib/action.ts

```diff
@@ -810,6 +810,18 @@ class Action {
 		analytics.event('ThemeSet', { id });
 	};
 
+	publish (objectId: string) {
+		C.ObjectPublish(S.Common.space, objectId, (message: any) => {
+			if (message.error.code) {
+				return;
+			};
```

Comment:
you can just skip brackets and make in online.
`if (message.error.code) return`

Reply:
We do not write one-line if clauses.